### PR TITLE
Menu bar formatting (stage 1)

### PR DESCRIPTION
**Pull request by leenlab2**
This pull request includes basic positioning and format changes to the menu bar.

## Motivation and Context

This pull request is part of a series that alters the layout of the Graph page. The details are outlined in Issue #1218.

## Your Changes

**Description**:

- Reduced size of the Courseography logo and altered position respectively
- Removed the black border around the menu bar and the navigation bar items
- Centered the navigation bar items
- Slightly reduced the size of navigation bar items and altered position respectively

**Type of change** (select all that apply):

- [x] Other: User Interface changes
## Testing

Manual testing in the browser.

## Questions and Comments (if applicable)

I wasn't really sure what type of change to label this with. Also, although I mentioned the issue I'm not sure how to link the PR to it. Finally, do we need to add tests for these layout changes?

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have verified that the CircleCI tests have
  passed.

### CODE REVIEW

**Comment by leenlab2** [comment]
Comparison screenshots to showcase changes:

Before
image

After
image

File: style/app.scss [hunk 1]
```diff
@@ -147,30 +146,28 @@ body
 #nav-links
 {
   list-style  : none;
-  min-width   : 687px;
-  padding-top : 10px;
+  min-width   : fit-content;
+  padding     : 11 0 0 0;
```
Comment (david-yz-liu) [inline style/app.scss:150]: Always use units with non-zero dimension values

File: style/app.scss [hunk 2]
```diff
@@ -147,30 +146,28 @@ body
 #nav-links
 {
   list-style  : none;
-  min-width   : 687px;
-  padding-top : 10px;
+  min-width   : fit-content;
```
Comment (david-yz-liu) [inline style/app.scss:149]: You can just delete this rule altogether

File: style/app.scss [hunk 3]
```diff
@@ -147,30 +146,28 @@ body
...
   height      : 50px;
   position    : absolute;
+  left        : 50%;
+  transform   : translate(-50%, 0);
 }
 
 #nav-links a
 {
   font-weight : normal;
+  color       : #FFFFFF;
```
Comment (david-yz-liu) [inline style/app.scss:162]: use lowercase letters in hex colour values

File: style/app.scss [hunk 4]
```diff
@@ -147,30 +146,28 @@ body
...
   text-align : center;
   display    : inline-block;
-  padding    : 0px 10px 0px 10px;
-}
-
-#nav-links > li > a
-{
-  color       : #ffffff;
-  text-shadow : 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000, 0 0 2px #000;;
+  padding    : 0px 25px 0px 25px;
```
Comment (david-yz-liu) [inline style/app.scss:170]: You can delete the units for the `0`s here

File: style/app.scss [hunk 5]
```diff
@@ -213,9 +210,10 @@ body
 
 #courseography-header
 {
-  width          : 280px;
-  height         : 50px;
-  padding-bottom : 3px;
+  width          : 200px;
+  height         : auto;
```
Comment (david-yz-liu) [inline style/app.scss:214]: You can delete this rule

File: style/app.scss [hunk 6]
```diff
@@ -213,9 +210,10 @@ body
...
+  padding-top    : 10px;
+  margin         : 0 10 0 30;
```
Comment (david-yz-liu) [inline style/app.scss:216]: See above comments about units

**Pull request merged**